### PR TITLE
fix(ftux): simplified status fetching logic

### PR DESCRIPTION
**Pull request by neurosnap**
design(ftux): updated copy and design elements

https://user-images.githubusercontent.com/1940365/233132910-c7bbf189-9d6b-4004-bef0-8b04fdc2c303.mov

### CODE REVIEW

**Review by madhuravius — COMMENTED** [review]
despite the title this is just a super PR of issues caught, right?

File: src/ui/pages/create-project-git.tsx [hunk 1]
```diff
@@ -358,7 +373,9 @@ export const CreateProjectAddKeyPage = () => {
   return (
     <div>
       <div className="text-center">
-        <h1 className={tokens.type.h1}>Deploy your code</h1>
+        <h1 className={tokens.type.h1}>
```
Comment (madhuravius) [inline src/ui/pages/create-project-git.tsx:376]: this is fine for now but i saw `Get ready to deploy your app on Aptible` repeated a few times, maybe worthy of a common onboarding header func if we ever change text

File: src/ui/pages/create-project-git.tsx [hunk 2]
```diff
@@ -1694,7 +1764,7 @@ export const CreateProjectGitStatusPage = () => {
   // if the user started the deployment process but left before
   // we could create an app deploy operation then we need to kick that
   // off again here
-  useEffect(() => {
+  /* useEffect(() => {
```
Comment (madhuravius) [inline src/ui/pages/create-project-git.tsx:1767]: this feels unintentional

Comment (neurosnap) [reply src/ui/pages/create-project-git.tsx:1767]: Intentional, too many side-effects.

**Comment by madhuravius** [comment]
LGTM so far, let me know when closer to final state

**Review by madhuravius — DISMISSED** [review]

**Review by madhuravius — APPROVED** [review]

**Pull request merged**